### PR TITLE
{cae}[foss/2023a] OpenFOAM 10 w/ CGAL 5.6 and ParaView 5.11.2

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm (

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 8 out of 8 (3 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/14e5ba254a132715f0e41e5616fa35fa for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
zen2-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/f2cef35e989106f117c976086ee149dd for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/ParaView/ParaView-5.11.2-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.11.2'
+versionsuffix = ''
```
Comment (boegel) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.11.2-foss-2023a.eb:5]: no need for this, that's the default

```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3127.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/e3e60e1df797cef78ee8baf10c125cdd for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Comment by paulmelis** [comment]
So now there's ParaView-5.11.2-foss-2023a.eb from this pull request, but also the fat-build of ParaView-5.11.1-foss-2022b.eb (with many more features, plus patches) from https://github.com/easybuilders/easybuild-easyconfigs/pull/18631. That doesn't seem ideal, as I would hope the 5.11.1 with more testing and features is slated for the next EB release. @lcniel any thoughts?

**Pull request merged**